WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -32,29 +32,38 @@ class SiteDesignContentCollectionViewController: CollapsableHeaderViewController
                 scrollableView.setContentOffset(.zero, animated: false)
             }
             sections = siteDesigns.categories.map { category in
-                SiteDesignSection(category: category, designs: siteDesigns.designs.filter { design in design.categories.map({$0.slug}).contains(category.slug)
-                })
+                SiteDesignSection(
```

Comment:
For recommended designs we should be able to to something similar by passing in a larger thumbnail size.